### ISSUE TITLE
Trying to use tasker makes Rotini crash

### ISSUE THREAD

**Issue by anordvall** [igorgladkov/rotini#34]
When I tried to use tasker to turn lights off when an app start, Rotini crashed.

Steps to reproduce:
1. Start Tasker
2. Add a profile with context Application and select e.g. Chrome
3. Select to add a new Task
4. Add a Plugin action
5. Select Project Rotini
6. Click on Configuration and select a Switch item
7. Click on Command and type in "OFF"
8. Select DONE

Result: Rotini crashes

I added my nickname to the report being sent.

Of course, I don't know if I've done this the right way or not as I haven't really seen any documentation of the tasker integration yet ;)

**Comment by anordvall** [comment]
Actually, it seems the crash will also occur after step 5 if you just click on CANCEL without adding any item and command.

**Comment by igorgladkov** [comment]
Fixed in 0.2.19
